.bazelci/presubmit.yml

```diff
@@ -7,6 +7,10 @@ tasks:
   ubuntu1804:
     shell_commands:
```

Comment:
You should no longer need cargo.

Reply:
I've gotten a PR through `rules_rust` to support hermetic testing with environment variables.